bitcoin/script/miniscript.cpp

```diff
@@ -253,8 +253,8 @@ size_t ComputeScriptLen(NodeType nodetype, Type sub0typ, size_t subsize, uint32_
     switch (nodetype) {
         case NodeType::PK_K: return subsize + 34;
         case NodeType::PK_H: return subsize + 3 + 21;
-        case NodeType::OLDER: return subsize + 1 + (CScript() << k).size();
-        case NodeType::AFTER: return subsize + 1 + (CScript() << k).size();
+        case NodeType::OLDER: return subsize + 1 + BuildScript(k).size();
```

Comment:
Unrelated to this PR but i wonder why we add `subsize` here. It must always be 0 so that's unnecessarily confusing. Same for all fragments without subs in this function.

Removing them on the PR following this comment: https://github.com/bitcoin/bitcoin/pull/24147#discussion_r796126519